### ISSUE TITLE
TADS support to-do list

### ISSUE THREAD

**Issue by PeterPiers** [ifrotz/iosfrotz#281]
I'm still to do proper extensive testing in TADS, but I think I can post a to-do list with the most obvious "issues" (not really issues at all, but "things which are probably the next in line to get around to). 

1 - Autocomplete doesn't work. If TADS games don't have an available dictionary like Inform games do, maybe the standard autocomplete dictionary iFrotz used before could be loaded instead? In fact, this might also be a good idea for those odd few Inform games where autocomplete doesn't work either.

2 - Hyperlinks don't work. Worryingly, though, there's no sign that there IS a hyperlink in place, so if I don't already know the story has hyperlinks I won't know that I'm missing out on functionality. That may be a severe issue, depending on whom you ask (if you ask me, it is).

3 - Graphics don't display. Similarly to above, there's also no placeholder for them. Different games behave differently; "Arrival" seems to crash while filling the screen with question marks when you get to the point where the game displays its title screen, whereas 1893 ran great with only a huge blank space where the graphic should be, Further testing into each specific case is required. But again, with no indication I'm missing out, I - as a player - do not know (and I should know) that I should instead play this game on a different interpreter.

4 - No sound (also on the TO-DO list for the Inform games!).

This would seem to be it for now. I'll update as necessary after proper testing (it'll be a while, mind).

It's worth noting that I've made sure that games which have multimedia files, in the form of either external resource files (similar to the .blb files in some old Inform games) or actual external image and sound files, also exist in a bundled form. TADS games for iFrotz do not have to look for external files - that'd be impossible anyway, given the iOS way of doing things, I gather. Neither do the users have to upload the resource files separately. If the user loads the bundled version of one of these games, the file will already have the necessary multimedia files. They may be in "resource file" or "image/sound file" format, but they will be there. I do wish I could do this with some ZCode games as well, but my attempts on that came to naught.

**Comment by KevinBlue18** [comment]
I agree with @PeterPiers some TADS games crash for me, especially A. Ninny’s  _Malaise_ a sequel to _Moist by Scartlet Herring_.
